.github/workflows/release.yml

```diff
@@ -3,36 +3,38 @@ name: Release
 on:
   milestone:
     types: [closed]
```

Comment:
might want to remove this in the future. i left it for now. i prefer the tag based approach personally